stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/reactions/ReactionsOverlayDialogFragment.kt

```diff
@@ -142,14 +143,16 @@ internal class ReactionsOverlayDialogFragment : DialogFragment() {
         )
     }
 
+    private class MessageItemWrapper(val messageListItem: MessageListItem.MessageItem) : Serializable
```

Comment:
I made `MessageItem` serializable now to use the wrapper

Reply:
Done